docs/units-and-global-variables.rst

```diff
@@ -22,8 +22,8 @@ unit and units are considered naively in the following way:
  * ``1 minutes == 60 seconds``
  * ``1 hours == 60 minutes``
  * ``1 days == 24 hours``
- * ``1 weeks = 7 days``
- * ``1 years = 365 days``
+ * ``1 weeks == 7 days``
```

Comment:
Could you change these and all the above to singular on the left? e.g. `1 week == 7 days`?

Plural for 1 seems to be bad 😉